ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17-periodics.yaml

```diff
@@ -74513,6 +74513,17 @@ periodics:
     job-release: "4.17"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-gcp-ipi-sno-etcd-encryption-arm-f14
+  reporter_config:
+    slack:
+      channel: '#aos-qe-logging'
```

Comment:
@anpingli This job shouldn't be reported to logging channel. You may have to update https://github.com/openshift/release/pull/61281/files#diff-dbad6f53b2f7eef7d2f43cc3ae6974f7b724c771667250c57827c0bf307698c6R35

Reply:
Done